app/views/maps/show.html.erb

```diff
@@ -1,4 +1,14 @@
 <% content_for :title do %>
+  <div class="pull-right dropdown">
```

Comment:
Hmm, was this missing? Or was it perhaps to be added in via JavaScript? Sorry, i'm a bit out of sync on this. Thank you!